plugins/periskop-backend/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@backstage/plugin-periskop-backend",
+  "version": "0.1.0",
```

Comment:
we usually start these at 0.0.0 and let the changeset bump it to 0.1.0 at `minor` release